src/cli/common.rs

```diff
@@ -522,3 +522,18 @@ pub fn report_error(e: &Error) {
         }
     }
 }
+
+pub fn non_fatal_error(error: crate::errors::Error, question: &str, default: bool, no_prompt: bool) -> Result<()> {
+    report_error(&error);
+    if no_prompt {
```

Comment:
Does this skipping also want an environment variable, or is `-y` sufficient?

Reply:
Currently, it just uses `-y`, but I could also add an environment variable.

Reply:
I have now added `RUSTUP_INIT_SKIP_CHECKING_EXISTENCE` to completely skip the check.